langwatch/src/server/tracer/opentelemetry.ts

```
@@ -901,52 +901,67 @@ type RecursiveRecord = {
   [key: string]: (RecursiveRecord & (string | {})) | undefined;
 };
 
-const keyValueToObject = (
+const isNumeric =(n: any) => !isNaN(parseFloat(n)) && isFinite(n);
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)